index.html

```diff
@@ -32,9 +32,9 @@
     <script src="bower_components/urijs/src/URI.min.js"></script>
     <script src="js/zap-common.js"></script>
     <script src="js/everything.js"></script>
-    <script src="https://apis.google.com/js/api.js?onload=handleClientLoad"></script>
+    
     <script src="bower_components/plyr/dist/plyr.js"></script>
-
+    <script src="js/api.js"></script>
```

Comment:
Well, I think I've figured out the problem 😩. The Google Client ID allows requests from certain domains only, the authentication request has to come from one of these domains only, the Zombie Browser uses file:// protocol, which in turn gets rejected from Google and Zombie receives 400 bad request response.
To bypass this error, I replaced this line:
`var indexHTMLURL = "file://" + path.join(__dirname, "..", "index.html");`
by
`var indexHTMLURL = "http://localhost/open-source/zen-audio-player.github.io/index.html";`
in `test/index.js` 
But now after this error is bypassed, another error appears 😫. The empty search form check, which applies a submit button press and waits for the `local site test` error message. Due to using the http protocol the function isFileProtocol will return false which won't show the error message `local site test`, thus the assert fails. I hope I didn't miss anything and I think it makes sense now 😄